server/graphql/v1/mutations/expenses.js

```diff
@@ -339,6 +340,9 @@ export async function payExpense(remoteUser, expenseId, fees = {}) {
     if (expense.paypalEmail === paymentMethod.name) {
       feesInHostCurrency.paymentProcessorFeeInHostCurrency = 0;
       await createTransactions(host, expense, feesInHostCurrency);
+    } else if (processorFeeInputed && processorFeeInputed > 0) {
+      // For paypal method manually recorded as paid.
+      await createTransactions(host, expense, feesInHostCurrency);
     } else {
       await payExpenseWithPayPal(remoteUser, expense, host, paymentMethod, feesInHostCurrency);
     }
```

Comment:
If we manually mark a PayPal expense as paid and set `0` for the processor fees, it will try to pay with PayPal by calling the API (`payExpenseWithPayPal`). Though fees are not supposed to be `0`, I'm not sure that it's a safe behavior - if for any reason host admins puts `0` in there we could end up double-paying expenses.

Reply:
I think we should consult @alanna with this.

Reply:
Feel free to ask her if we should let users put `0` for fees.

In the same time this code should be fixed cause the current behavior is unexpected.

Reply:
Followed up @alanna [response](https://github.com/opencollective/opencollective/issues/1976#issuecomment-558495730), I've update this solution, I introduced `manuallyPaypalMethod`  to `payExpense` mutation parament as Boolean, the method uses the argument to determine how to handle the expense payment.